### DISCUSSION TITLE
students

### DISCUSSION THREAD

**Discussion by 106-220026** [community/community#146265] [GitHub Education]
### Hi everyone,

i am a students i wanna complete my two factor aut but my country pakistan not in the list

**Comment by swastiswagat** [comment]
You can use the Google Authenticator app. Simply scan the QR code or enter the key provided to set it up. When you want to log in, use the code displayed in the app. Please note that the code changes every 30 seconds. For more detailed instructions, you can also check tutorials on YouTube.

**Reply by 106-220026** [reply]
can you send me link of yt

**Comment by 106-220026** [comment]
a lot of thanks dear ....love you
________________________________
